lib/rubocop/cop/sorbet/prefer_sorbet_over_contracts.rb

```diff
@@ -0,0 +1,114 @@
+# frozen_string_literal: true
+
+# Disallows the use of Contracts and converts Contracts to Sorbet signatures
+#
+# For example:
+#
+#  Contract Integer, Boolean => String
+#  def self.cool_thing(number, bool)
+#    return false
+#  end
+#
+#  Will be converted to:
+#
+#  sig { params(number: Integer, bool: T::Boolean).returns(String) }
+#  def self.cool_thing(number, bool)
+#    return false
+#  end
+#
+# Additonally `extend T::Sig` will be added when necessary.
+#
```

Comment:
This comment should be attached to the `PreferSorbetOverContracts` class. Also, you can use [fenced code blocks](https://help.github.com/en/github/writing-on-github/creating-and-highlighting-code-blocks#syntax-highlighting) for a better look.